### PR TITLE
Enable CI unit-tests

### PR DESCRIPTION
**Pull request by sturivny**

### CODE REVIEW

**Comment by sturivny** [comment]
@bocekm Working on it

**Comment by bocekm** [comment]
@sturivny, please rebase.

**Comment by sturivny** [comment]
@bocekm please review.

File: .packit.yaml [hunk 1]
```diff
@@ -16,11 +16,13 @@ jobs:
     - epel-7-x86_64
   trigger: pull_request
 - job: tests
-  trigger: pull_request
   metadata:
+    owner: "@oamg"
+    project: convert2rhel
```
Comment (bocekm) [inline .packit.yaml:21]: Is this necessary for tests? It seems like it's just for copr.

File: convert2rhel/main.py [hunk 2]
```diff
@@ -92,7 +92,7 @@ def main():
         # restart system if required
         utils.restart_system()
 
-    except (Exception, SystemExit, KeyboardInterrupt), err:
+    except (Exception, SystemExit, KeyboardInterrupt) as err:
```
Comment (bocekm) [inline convert2rhel/main.py:95]: Using "," is necessary for python 2.4 compatibility (RHEL 5). I believe we don't need the code to be Py3-compatible as of yet. Maybe later if we go for conversions to RHEL 8.

File: convert2rhel/pkghandler.py [hunk 3]
```diff
@@ -196,7 +195,6 @@ def get_installed_pkg_objects(name=""):
     """Return list with installed package objects. The packages can be
     optionally filtered by name.
     """
-    yum_base = yum.YumBase()
```
Comment (bocekm) [inline convert2rhel/pkghandler.py:199]: Is there any problem with the yum module? Without this line, the `yum_base` will be uninitialized.

Comment (sturivny) [reply convert2rhel/pkghandler.py:199]: Returned it back as it was

File: convert2rhel/main.py (hunk 2 above)
Comment (sturivny) [reply convert2rhel/main.py]: Returned it back as it was

File: .packit.yaml (hunk 1 above)
Comment (sturivny) [reply .packit.yaml]: Removed it

**Comment by sturivny** [comment]
@bocekm Thanks for reviewing :)
Updated PR

**Comment by sturivny** [comment]
> It looks good. But could be please undelete the _convert2rhel/unit_tests/pkghandler_test.py_ file?

Done :)

**Pull request merged**